src/components/report-technical-issue/template.njk

```diff
@@ -0,0 +1,2 @@
+<a class="govuk-link {{ params.classList }}" target="_blank" href="/contact/problem_reports_nonjs?service={{ params.serviceCode | default("unknown") }}"
```

Comment:
Is `rel="noopener noreferrer"`needed as we're opening in a new tab? I think the answer is no but thought I'd raise anyway.

Reply:
It's important if you're linking to external pages, this one is still inside HMRC therefore the security issue doesn't really exist.  I think the non-js version picks up information from the referrer and adds it to the information logged with the ticket, if they may do that then I'd see a benefit in passing that information through.